integration-tests/scaffolding/extrinsicHelpers.ts

```diff
@@ -10,6 +10,14 @@ import { connect, connectPromise } from "./apiConnection";
 import { DispatchError, Event, SignedBlock } from "@polkadot/types/interfaces";
 import { IsEvent } from "@polkadot/types/metadata/decorate/types";
 import { ItemizedStoragePageResponse, MessageSourceId, PaginatedStorageResponse, SchemaId } from "@frequency-chain/api-augment/interfaces";
+import BN from "bn.js";
+
+export type VestingSchedule = {
+    start: number;
+    period: number;
+    periodCount: number;
+    perPeriod: BN;
```

Comment:
Not blocking and Wil can help.
```suggestion
    perPeriod: bigint;
```

Reply:
Thanks for making the change.